client/galaxy/scripts/apps/login.js

```diff
@@ -0,0 +1,39 @@
+
+var jQuery = require( 'jquery' ),
+    $ = jQuery,
+    PANEL = require( 'layout/panel' ),
+    _l = require( 'utils/localization' ),
+    PAGE = require( 'layout/page' );
+
+window.app = function app( options, bootstrapped ){
```

Comment:
Why do we need globals here. Are these changes necessary for proper packing. This class looked straightforward before: https://raw.githubusercontent.com/galaxyproject/galaxy/dev/client/galaxy/scripts/mvc/app/app-login.js.